scipy/stats/_unuran/unuran.pxd

```diff
@@ -90,7 +90,7 @@ cdef extern from "unuran.h" nogil:
 
     int unur_gen_resetsub(UNUR_GEN* generator)
 
-    ctypedef double (*_unur_urng_new_sampleunif_ft)(void* state)
+    ctypedef double (*_unur_urng_new_sampleunif_ft)(void* state) except *
```

Comment:
This is just workaround. This should be fixed or in cython or in numpy.

Reply:
Removed workaround since there is a PR with the fix - https://github.com/cython/cython/pull/5386